cmd/calyptia/create_core_instance_gcp.go

```diff
@@ -138,11 +149,26 @@ func newCmdCreateCoreInstanceOnGCP(config *config, client gcp.Client) *cobra.Com
 	fs.StringVar(&sshUser, "ssh-user", "", "GCP SSH user to use for the instance. (default is no user)")
 	fs.StringVar(&sshKeyPath, "ssh-key", "", "SSH Key path to use for the instance. (default is no key)")
 	fs.StringVar(&network, "network", "default", "GCP Network")
-	fs.BoolVar(&useTestImages, "use-test-images", false, "Use GCP test images instead of released channel (only for testing/development).")
 
 	return cmd
 }
 
+func getGitHubToken() (string, error) {
+	token := os.Getenv("CALYPTIA_GITHUB_TOKEN")
```

Comment:
Why not just `GITHUB_TOKEN` which is pretty standard?

Reply:
yep, can be, its just because others have the calyptia prefix, but i can remove

Reply:
@gabrielbussolo yes, lets stick to GITHUB_TOKEN here.